ui/userdetails/src/commonMain/kotlin/dev/burnoo/demo/listapp/ui/userdetails/composables/UserDetailsContent.kt

```diff
@@ -0,0 +1,25 @@
+package dev.burnoo.demo.listapp.ui.userdetails.composables
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import dev.burnoo.demo.listapp.core.designsystem.composables.Loader
+import dev.burnoo.demo.listapp.core.designsystem.composables.TryAgain
+import dev.burnoo.demo.listapp.core.ui.collectAsStateWithLifecycle
+import dev.burnoo.demo.listapp.ui.userdetails.UserDetailsUiState
+import dev.burnoo.demo.listapp.ui.userdetails.UserDetailsViewModel
+
+@Composable
+internal fun UserDetailsContent(viewModel: UserDetailsViewModel, onGoBack: () -> Unit) {
+    val uiState by viewModel.uiState.collectAsStateWithLifecycle()
```

Comment:
Does it differ from the usual `StateFlow.collectAsState`, other than it being KMP-compatible?

Reply:
It does not collecting flow values when app is in the background - [here](https://medium.com/androiddevelopers/consuming-flows-safely-in-jetpack-compose-cde014d0d5a3) is a nice article about it.

Reply:
Nice, thanks.